src/shared/sd-dns-resolver.c

```diff
@@ -0,0 +1,362 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "dns-resolver-internal.h"
```

Comment:
we don't separate these out usually.

(we only separate out the sd-*.h includes, i.e. our public API .h includes)